src/main/java/net/imagej/ops/topology/EulerCharacteristic26N.java

```diff
@@ -89,7 +89,8 @@ public Double compute1(RandomAccessibleInterval<B> interval) {
      * @param z         Location of the neighborhood in the 3rd spatial dimension (z)
      * @return the index of the Δχ value for this configuration of voxels
      */
-    private int neighborhoodEulerIndex(final RandomAccess<B> access, final long x, final long y, final long z) {
+    public static <B extends BooleanType<B>> int neighborhoodEulerIndex(final RandomAccess<B> access, final long x,
+                                                                        final long y, final long z) {
```

Comment:
Note that this does not conform to ImageJ code style. We should probably create an approximation of the code style rules for IntelliJ at some point.